src/server.js

```diff
@@ -25,6 +25,7 @@ Server.prototype.listen = function(port, host) {
     var client = new Client(true);
     client._end = client.end;
     client.end = function end(endReason) {
+      endReason='{"text":"'+endReason+'"}';
```

Comment:
Wait so does this work in 1.8 ? Can we send an 1.8 client { "text": "Disconnected" } ?